publications.html

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title: Publications
+---
+
+
+<div class="colorlib-intro colorlib-bg-white">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-10 col-md-offset-1 colorlib-heading animate-box">
+                <h2>IEEE NITK - Papers submitted to coference</h2>
```

Comment:
Have the heading as `List of Papers Submitted to Conferences by IEEE NITK Members`. Same as before.